man/man8/puppet-lookup.8

```diff
@@ -1,7 +1,13 @@
 .\" generated with Ronn/v0.7.3
 .\" http://github.com/rtomayko/ronn/tree/0.7.3
 .
+<<<<<<< HEAD
 .TH "LOOKUP" "8" "September 2019" "Puppet, Inc." "Puppet manual"
+||||||| merged common ancestors
+.TH "PUPPET\-LOOKUP" "8" "August 2019" "Puppet, Inc." "Puppet manual"
+=======
+.TH "PUPPET\-LOOKUP" "8" "September 2019" "Puppet, Inc." "Puppet manual"
+>>>>>>> upstream/5.5.x
```

Comment:
Looks like conflict isn't resolved.

Reply:
I must have forgotten to save the file from my editor.  I redid the mergeup, I think it looks OK now.